go.mod

```diff
@@ -2,15 +2,15 @@ module mergemock
 
 go 1.17
 
-replace github.com/ethereum/go-ethereum => github.com/lightclient/go-ethereum v1.10.12
+replace github.com/ethereum/go-ethereum => github.com/MariusVanDerWijden/go-ethereum v1.8.22-0.20220204213217-0569213dc403
```

Comment:
Kind of odd version change, an outdated `master` branch I suppose, with the git commit updating it to the latest merge-ready version? (https://github.com/MariusVanDerWijden/go-ethereum/pull/39)

Reply:
Yeah I originally set it to Marius' branch `merge-kiln` and I think `go mod tidy` may have replaced it with the revision instead.